composer.json

```diff
@@ -25,7 +27,10 @@
         "ext-mbstring": "*",
         "ext-curl": "*",
         "composer/ca-bundle": "~1.0",
-        "symfony/options-resolver": "~2.7|~3.0"
+        "symfony/options-resolver": "~2.7|~3.0",
+        "php-http/httplug": "~1.1",
+        "php-http/curl-client": "~1.7",
```

Comment:
I strongly agree with you, and I thought a lot about this when looking at the official documentation of Httplug that clearly says [this is the way to go](http://docs.php-http.org/en/latest/httplug/library-developers.html) for library developers. I chosen to specify a client (cURL in this case) because I thought @dcramer would not approve to ship an SDK without a HTTP client. If we all agree on this, I will edit the PR accordingly.